recipes/lovely-numpy/meta.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+  name: lovely-numpy
+  version: 0.1.3
+source:
+  sha256: 0786c7226a59c8a676904409dce0807cc0192d92f4e4631c6ef9016de8cd9107
+  url: https://files.pythonhosted.org/packages/27/5c/93ab6fbe0deeada74cf8cf22fd460552c3d8bd98f450c6f90e4ca3a3d916/lovely-numpy-0.1.3.tar.gz
```

Comment:
The UUIDs in this URL will change with every release and cannot be templated. Grayskull will put in the correct one.